pandas/core/generic.py

```diff
@@ -2966,7 +2967,10 @@ def to_latex(
             The LaTeX caption to be placed inside ``\caption{}`` in the output.
 
             .. versionadded:: 1.0.0
-
+        short_caption : str, optional
+            The LaTeX short caption.
+            Full caption output would look like this:
+            ``\caption[short_caption]{caption}``.
```

Comment:
needs a versionadd tag

Reply:
Should I put this?
```
.. versionadded:: 1.2.0
```

I was not sure about the version.